explainer.md

```diff
@@ -79,136 +79,152 @@ A **VideoTrackWriter** converts a WritableStream of DecodedVideoFrame into a Med
 ### Example of decode for low-latency live streaming or cloud gaming 
 
 ```javascript
-const transport = ...;  // Source of muxed/serialized messsages
-// App-specific serialization/containerization code not provided by browser
-const demuxer = ...;  // Transforms to demuxed/deserialized frames
-const audioBuffer = ...;  // TransformStream that buffers frames
-const videoBuffer = ...;
+// The app provides ReadableStreams of encoded audio and video
```

Comment:
How do you feel about the term "framed bitstream chunk"?

Reply:
Sounds good.